mgnify_pipelines_toolkit/constants/bgc_descriptions/antismash_glossary_7.1.x.md

```diff
@@ -0,0 +1,147 @@
+#Protocluster Types
+
+antiSMASH uses some abbreviations internally and in outputs to refer to the different
+types of secondary metabolite clusters detected.
+
+There are three strictness levels for cluster detection: strict, relaxed, and loose.
+Each level includes the more strict levels.
+The types of cluster within each ruleset are given below:
+
+##Current Types
+
+
+### Strict
```

Comment:
I would have this in the constants of the repo or in this same script, a .md is useful for documentation but for this use case it is an overkill.

This .md belongs in the MGnify docs

Reply:
It took me reasonable amount of time to write a parser. For every antismash release you'll have re-generate those.. I don't think that is overkill. Or at least add that parser to DBs preparation pipeline. We can't hardcode those to script, I absolutely against it. That .md doesn't belong to MGnify, it is from antismash https://github.com/antismash/documentation/blob/880e7d9d9f60a7d122aa0a1a9e77d37db29200a0/docs/glossary.md